packages/Sandblocks-Core/SBBlock.class.st

```diff
@@ -150,6 +150,27 @@ SBBlock >> attachDecorator: aDecorator [
 	self updateSteppingDecorators
 ]
 
+{ #category : #'as yet unclassified' }
+SBBlock >> attractionStrengthTo: aMorph [
+	
+	(self isAttractedTo: aMorph)
+		ifTrue: [^  0.01 * (aMorph center - self center) r]
+		ifFalse: [^ 0.0]
```

Comment:
best to move the `^` outside the condition :) (same for most ifTrue:ifFalse: below)